src/BinSkim.Sdk/MetadataConditions.cs

```diff
@@ -15,6 +15,8 @@ public static class MetadataConditions
         public static readonly string ImageIsBootBinary = SdkResources.MetadataCondition_ImageIsBootBinary;
         public static readonly string ImageIs64BitBinary = SdkResources.MetadataCondition_ImageIs64BitBinary;
         public static readonly string ElfNotBuiltWithGcc = SdkResources.MetadataCondition_ElfNotBuiltWithGCC;
+        public static readonly string ElfNotBuiltWithGccV8OrLater = SdkResources.MetadataCondition_ElfNotBuiltWithGccV8OrLater;

```

Comment:
>ElfNotBuiltWithGccV8OrLater [](http://example.com/codeflow?start=38&length=27)

move near the end to make the length in crescent order :)

Reply:
fixed. This style is cool :)